.github/workflows/lintmd5.txt

```diff
@@ -0,0 +1 @@
+f7657762cd9129ae96dfaecafd3f5df9  -
```

Comment:
what is this for?

Reply:
It was a lint check to ignore false positive "new" results.
The original instructions we received targeted an Observer Microservice in a different repository. I was not sure whether this is still the goal, so I just removed the lint check from the flow-go repo.